modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/components/PodcastCover.kt

```diff
@@ -0,0 +1,75 @@
+package au.com.shiftyjelly.pocketcasts.endofyear.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Matrix
+import androidx.compose.ui.graphics.drawscope.withTransform
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.components.PodcastImage
+
+private const val PodcastCoverRotationAngle = -30f
+private const val PodcastCoverSkew = 0.45f
+
+@Composable
+fun PodcastCover(
```

Comment:
It uses a card elevation instead of the gradients provided. Again, this will be improved in a future PR.